app/lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -157,7 +157,7 @@ def input_text_form_group(attribute, options)
       classes = ['govuk-form-group']
       classes << 'govuk-form-group--error' if error?(attribute, options)
       content_tag :div, class: classes.join(' ') do
-        label = label(attribute, options[:label], class: 'govuk-label', for: attribute)
+        label = options[:label] == 'no_label' ? '' : label(attribute, options[:label], class: 'govuk-label', for: attribute)
```

Comment:
I will update and add the spec now.

Reply:
The reason i use `label: 'no-label'`  because if i used `label: nil`  this will break the default behaviour of `govuk_text_field(:email)`  where you would get a label Email.  In this example   options[:label] is set to `nil`.   I wanted the user of this method to force not to display a label.

Reply:
`label: nil` works if you change the new condition in the form builder to : 

```ruby
options.key?(:label) && options[:label].nil? ? '' : ...
```

Otherwise, we could use `false` instead of `nil` or `'no_label'`. And also change it for `hint`
`{ hint: false, label: false }`

Or change both to `:hide`
`{ hint: :hide, label: :hide }`

I think that it would make things confusing if we use different parameters to hide the label and the hint

Reply:
updated to use `label: nil` to not display a label, the same as to not display a hint